### DISCUSSION TITLE
MeshoptCompression documentation

### DISCUSSION THREAD

**Discussion by gnansai** [donmccurdy/glTF-Transform#709] [General]
I think import  of "reorder" and "quantize" should be in the example code. It was showing error "ReferenceError: reorder is not defined".

```
import { reorder } from "@gltf-transform/functions";
import { quantize } from "@gltf-transform/functions";
```

![image](https://user-images.githubusercontent.com/25223402/197346524-4abb5064-55d9-405b-ab1a-7273a9c0f2d6.png)

**Comment by donmccurdy** [comment]
Thanks @gnansai! I've updated the  documentation:


https://gltf-transform.donmccurdy.com/classes/extensions.meshoptcompression.html
